src/cli/abrt-cli-core.c

```diff
@@ -148,13 +153,18 @@ void initialize_polkit_agent(void)
     }
 
     g_object_unref(subject);
+#else
+    printf("Polkit support is currently disabled\n");
```

Comment:
Maybe we should rather use the common logging functions: https://github.com/abrt/libreport/blob/master/src/include/internal_libreport.h#L578

I propose to log this message as info message. IMHO it is not necessary to issue the message every time a user runs abrt-cli. Thoughts?